### PR TITLE
Update create-database-scoped-credential-transact-sql.md

### PR DESCRIPTION
**Pull request by macarv-ms**
typo in this document where it says, "Shared Access Signature cannot be used with Polybase." It should instead say, "Shared Access Signature can be used with Polybase."

### CODE REVIEW

**Comment by Jak-MS** [comment]
@VanMSFT 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by VanMSFT** [comment]
Thanks, @macarv-ms! This looks correct, but let's have @WilliamDAssafMSFT confirm.

**Comment by WilliamDAssafMSFT** [comment]
Thanks, @macarv-ms, nice catch. This is a good reason why we don't put tech specs in Example text, this text is 6+ years old and clearly has changed. It'd be best to just remove this sentence.

File: docs/t-sql/statements/create-database-scoped-credential-transact-sql.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ CREATE DATABASE SCOPED CREDENTIAL AppCred WITH IDENTITY = 'Mary5',
 
 ### B. Creating a database scoped credential for a shared access signature
 
-The following example creates a database scoped credential that can be used to create an [external data source](../../t-sql/statements/create-external-data-source-transact-sql.md), which can do bulk operations, such as [BULK INSERT](../../t-sql/statements/bulk-insert-transact-sql.md) and [OPENROWSET](../../t-sql/functions/openrowset-transact-sql.md). Shared Access Signatures cannot be used with PolyBase in SQL Server, APS or Azure Synapse Analytics.
+The following example creates a database scoped credential that can be used to create an [external data source](../../t-sql/statements/create-external-data-source-transact-sql.md), which can do bulk operations, such as [BULK INSERT](../../t-sql/statements/bulk-insert-transact-sql.md) and [OPENROWSET](../../t-sql/functions/openrowset-transact-sql.md). Shared Access Signatures can be used with PolyBase in SQL Server, APS or Azure Synapse Analytics.
```
Comment (WilliamDAssafMSFT) [inline docs/t-sql/statements/create-database-scoped-credential-transact-sql.md]:
```suggestion
The following example creates a database scoped credential that can be used to create an [external data source](../../t-sql/statements/create-external-data-source-transact-sql.md), which can do bulk operations, such as [BULK INSERT](../../t-sql/statements/bulk-insert-transact-sql.md) and [OPENROWSET](../../t-sql/functions/openrowset-transact-sql.md). 

```

**Comment by WilliamDAssafMSFT** [comment]
Thanks again @macarv-ms. 
Your commit will be merged and this article update is in flight. After the updated article is published, GitHub will list you as an article contributor. 

#sign-off

**Pull request merged**